docs/CONTRIBUTING.md

```diff
@@ -37,16 +37,18 @@ You'll need to generate mocks if an existing endpoint method is modified or a ne
 mockgen -source=example_resource.go -destination=mocks/example_resource_mocks.go -package=mocks
 ```
 
-## Adding API changes that are still behind a feature flag
+## Adding API changes that are not generally available
 
-On top of your code changes, or anywhere visible, add a comment that reads like this:
+In general, beta features should not be merged/released until generally available. However, the maintainers recognize almost any reason to release beta features on a case-by-case basis. These could include: partial customer availability, software dependency, or any reason short of feature completeness.
```

Comment:
```suggestion
In general, beta features should not be merged/released until generally available (GA). However, the maintainers recognize almost any reason to release beta features on a case-by-case basis. These could include: partial customer availability, software dependency, or any reason short of feature completeness.
```

I like how this reads! How would folks reach out to the maintainers? By opening a PR?

Reply:
I meant in the PR in question, itself, I think... but also by anything we choose since it's more or less incumbent upon us as maintainers to enforce this.